### PR TITLE
Use the new derive ID convention

### PR DESCRIPTION
**Pull request by kc1212**
The way we spawn contracts and derive ID was changed in master. Update
it so that the template compiles again.

Closes https://github.com/dedis/cothority_template/issues/41

### CODE REVIEW

**Comment by kc1212** [comment]
putting this back to WIP because we need a decision on versioning

**Comment by jeffallen** [comment]
v3 is not coming anytime soon, so let's just put master-$DATE versions in here for now. I'll continue making the releases as needed, and add this step to my notes.

File: omniledger/keyvalue.go [hunk 1]
```diff
@@ -34,17 +47,18 @@ func ContractKeyValue(cdb ol.CollectionView, inst ol.Instruction, cIn []ol.Coin)
 			return
 		}
 
-		log.Printf("Created new keyvalue at %x", inst.DeriveID(ContractKeyValueID))
+		instID := inst.DeriveID("")
+		log.Printf("Created new keyvalue at %x", instID)
```
Comment (ineiti) [inline omniledger/keyvalue.go]: remove `log.Printf`

**Review by ineiti — DISMISSED** [review]
looks good.

File: omniledger/keyvalue.go (hunk 1 above)
Comment (kc1212) [reply omniledger/keyvalue.go]: thanks, fixed

**Pull request merged**